go/mysql/query.go

```diff
@@ -448,6 +451,7 @@ func (c *Conn) ReadQueryResult(maxrows int, wantfields bool) (*sqltypes.Result,
 				more = (packetOk.statusFlags & ServerMoreResultsExists) != 0
 				result.SessionStateChanges = packetOk.sessionStateData
 				result.StatusFlags = packetOk.statusFlags
+				result.Info = packetOk.info
```

Comment:
I would have expected the `if` condition here rather than in `parseOKPacket`.

Reply:
I went back and forth with it. It felt wasteful to move the `info` into the `packetOK` struct if it's not going to be used anywhere.

Reply:
Is packetOk.info completely unused right now? In which case 👍